src/app/zap-templates/zcl/data-model/chip/time-format-localization-cluster.xml

```diff
@@ -55,6 +55,7 @@ limitations under the License.
       may have differing preferences for how dates and times are conveyed. As such, Nodes that visually
       or audibly convey time information need a mechanism by which they can be configured to use a
       user’s preferred format.</description>
+    <globalAttribute side="either" code="0xFFFD" value="2"/>    
```

Comment:
Why is this change here?  Spec has 1 as the cluster revision for this cluster, as far as I can tell.

Reply:
Not sure why this was added here